scripts/spack_packages/raja/package.py

```diff
@@ -274,9 +274,9 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
 
                 cfg.write(cmake_cache_string("BLT_CXX_STD", "c++14"))
             else:
-                cuda_release_flags = "-O3 -Xcompiler -Ofast -Xcompiler -finline-functions -Xcompiler -finline-limit=20000"
```

Comment:
If I remember correctly, keeping this host compiler flag allows some of the sequential reducer reset tests to pass. I don't think we figured out why, but @trws had some guesses as to why this occurs.

Reply:
I vaguely recall that. But, it would only be needed for the GNU compiler, correct? Let's see if the CI checks pass with this change.

Reply:
@rchen20 i made the spack package logic account for non-gnu and gnu compilers. hopefully, this will pass now.